### PR TITLE
Added logging to Submissions controller and service

### PR DESCRIPTION
**Pull request by harishpaladugu13**
[AB#456955](https://dev.azure.com/defragovuk/RWD-CPR-EPR4P-ADO/_workitems/edit/456955)

### CODE REVIEW

File: src/EPR.CommonDataService.Api/Controllers/SubmissionsController.cs [hunk 1]
```diff
@@ -1,28 +1,34 @@
+#nullable disable
```
Comment (taseakpoexternal) [inline src/EPR.CommonDataService.Api/Controllers/SubmissionsController.cs:1]: why have you disabled nullability? Its enabled for a reason!

File: src/EPR.CommonDataService.Core/Services/SubmissionsService.cs [hunk 2]
```diff
@@ -1,9 +1,15 @@
+#nullable disable
```
Comment (taseakpoexternal) [inline src/EPR.CommonDataService.Core/Services/SubmissionsService.cs:1]: here too

File: src/EPR.CommonDataService.Api/Controllers/SubmissionsController.cs (hunk 1 above)
Comment (harishpaladugu13) [reply src/EPR.CommonDataService.Api/Controllers/SubmissionsController.cs:1]: We have new injections of Logger and Config.
it's complaining about possible null value for config.

There are no alerts anywhere in the code

File: src/EPR.CommonDataService.Core/Services/SubmissionsService.cs (hunk 2 above)
Comment (harishpaladugu13) [reply src/EPR.CommonDataService.Core/Services/SubmissionsService.cs:1]: We have new injections of Logger and Config.
it's complaining about possible null value for config.

There are no alerts anywhere in the code

File: src/EPR.CommonDataService.Api/Controllers/SubmissionsController.cs (hunk 1 above)
Comment (harishpaladugu13) [reply src/EPR.CommonDataService.Api/Controllers/SubmissionsController.cs]: removed

**Review by saywasingh-eviden — APPROVED** [review]

**Review by taseakpoexternal — APPROVED** [review]

**Pull request merged**